apps/build.gradle

```diff
@@ -1,136 +1,83 @@
 /* will produce a fat jar containing all html/js dist in public_html/apps*/
```

Comment:
this file is greatly simplified as we don't need to think about the javascript, nx does all that